sphinx-doc/module-hoomd-operation.rst

```diff
@@ -0,0 +1,19 @@
+hoomd.operation
+---------------
+
+.. rubric:: Overview
+
+.. py:currentmodule:: hoomd.operation
+
+.. autosummary::
+    :nosignatures:
+
+    Analyzer
```

Comment:
We should also document `Operation` and `Compute`. Were you planning this in another PR?

Reply:
Operation yes, compute no. I will add it now.